container/es-releases.json

```diff
@@ -1,16 +1,25 @@
 {
   "release": {
-       "2.0.1.0" :{
+       "2.0.1.0" : {
           "sparkversion": "2.4.6",
            "sparkclient": "repo1.maven.org/maven2/com/ibm/event/ibm-db2-eventstore-client-spark-2.4.6/2.0.1.0/ibm-db2-eventstore-client-spark-2.4.6-2.0.1.0.jar",
            "hadoopversion":"2.6",
-           "scopt211version":"3.5.0"
+           "scopt211version":"3.5.0",
```

Comment:
I know this was here before but what's this used for ?  I didn't see us consuming this for any versioning ?